NewSpring/ApollosProject
# Upgrades to 8.1.2

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates our repo to version 8.1.2. This allows us to have sms login, onboarding, and content personalization.
### How do I test this PR?
1. yarn
2. yarn start
3. Make sure all new features are working as intended and make sure all old features still work.
---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3705](https://newspring.atlassian.net/browse/DEV-3705)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
